action.yaml

```diff
@@ -19,23 +20,38 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - uses: robinraju/release-downloader@v1.3
-      with:
-        repository: "fischerscode/tagger"
-        fileName: "tagger.linux"
-        out-file-path: ${{ github.action_path }}
-        tag: v0.1.0
-    - run: |
-        mv ${{ github.action_path }}/tagger.linux ${{ github.action_path }}/tagger
-        chmod +x ${{ github.action_path }}/tagger
-      shell: bash
-    - run: ${{ github.action_path }}/tagger move -s -p '${{inputs.prefix}}' ${INPUTS_TAG##*/}
+    - name: Collect info
+      id: info
+      run: |
+        if [ "${{ runner.os }}" = "Windows" ]; then
+          bin=${{ runner.temp }}\tagger.exe
```

Comment:
I had [this](https://github.com/fischerscode/action-test/actions/runs/4807627034/jobs/8556620610) run failing. 

It's probably due to the '\\' in windows paths. I think we need something like this:
https://github.com/fischerscode/action-test/blob/e336efcd5eb3782aa6aaa51bf24c9b380b363366/.github/workflows/test2.yaml#L7-L22

Reply:
Fixed. Thanks for sharing the example! Please review again 🙏